openmetadata-docs/content/deployment/docker/volumes.md

```diff
@@ -2,52 +2,115 @@
+Open the file `docker-compose.yml` downloaded from the Release page [Link](https://github.com/open-metadata/OpenMetadata/releases/download/x.x.x-release/docker-compose.yml) .
 
-```yaml
-version: 3.8
+```commandline
+version: "3.9"
 services:
-  db:
-    image:mysql
-    restart:always
+  mysql:
+    container_name: openmetadata_mysql
+    image: openmetadata/db:0.11.5
+    restart: always
     environment:
-       MYSQL_ROOT_PASSWORD: root
-       MYSQL_DATABASE: test_db
-    ports:
-      - "3306:3306"
+      MYSQL_ROOT_PASSWORD: password
+    expose:
+      - 3306
     volumes:
-      - db_data:/var/lib/mysql
-  volumes:
-    db_data
+     - /opt/openmetadata/db:/var/lib/mysql
+    networks:
+      app_net:
+        ipv4_address: 172.16.240.10
 ```
+## Volumes for PostgreSQL container:
+Following are the changes we have to do while mounting the directory for postgressql in OpenMetadata
+- Create a directory to keep your PostgresSQL data or files in the host machine
+```commandline
+mkdir -p /opt/openmetadata/db
+```
+- Update or add the volume in the docker-compose.yml file
+Open the file `docker-compose.yml` downloaded from the Release page [Link](https://github.com/open-metadata/OpenMetadata/releases/download/x.x.x-release/docker-compose.yml) .
 
+```commandline
+version: "3.9"
+services:
+ postgressql:
+    container_name: openmetadata_postgresql
+    image: openmetadata/postgresql:0.11.5
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+    expose:
+      - 5432
+    networks:
+      app_net:
+        ipv4_address: 172.16.240.10
+    volumes:
+     - /opt/openmetadata/db:/var/lib/postgressql
+    networks:
+      app_net:
+        ipv4_address: 172.16.240.10
+```
 
-Here, the first field is a unique name of the volume on a host machine. The second field is the path in the container.
-
-Following are the changes we have to do while mounting the volume for ingestion in OpenMetadata
-
-## Update or add the volume in the docker-compose.yml file
-
-Open the file `docker-compose.yml` downloaded from the Release page.
-
-First, define the volumes at the top level of the file. Example:
-
+## Volumes for ingestion container
+Following are the changes we have to do while mounting the directory for ingestion in OpenMetadata. Here we will maintaing different directory for dag_generated_configs, dags and secrets.
+- Create a directory to keep your ingestion data or files in the host machine
 ```commandline
-volumes:
-  ingestion-volume-dag-airflow:
-  ingestion-volume-dags:
-  ingestion-volume-tmp:
+mkdir -p /opt/openmetadata/dag_config
+mkdir -p /opt/openmetadata/dags
+mkdir -p /opt/openmetadata/secrets
 ```
+- Update or add the volume in the docker-compose.yml file
+Open the file `docker-compose.yml` downloaded from the Release page [Link](https://github.com/open-metadata/OpenMetadata/releases/download/x.x.x-release/docker-compose.yml) .
```

Comment:
The link is not a valid one! This will fail when use clicks on it to navigate!

Reply:
Done